src/utils/restUtils.ts

```diff
@@ -111,16 +126,18 @@ export enum HttpStatus {
     SERVICE_UNAVAILABLE = "Service Unavailable",
 }
 
-export const serverRequest = (method: string, urlPath: string, body: string|null|FormData, contentType?: string) => {
+export const serverRequest = (method: string, urlPath: string, body: string|null|FormData, contentType?: string, isSoknadApi?: boolean) => {
     const OPTIONS: RequestInit = {
         headers: getHeaders(contentType),
         method: method,
         credentials: determineCredentialsParameter(),
         body: body ? body : undefined
     };
 
+    const url = isSoknadApi ? getSoknadApiUrl() + urlPath : getApiBaseUrl() + urlPath
```

Comment:
Er det fare for at dette feiler i noen nettlesere dersom `isSoknadApi` er undefined?

Reply:
Ternary conditions sjekker bare truthy / falsy expressions, så ```undefined == false```. Bør ikke være noe problem i IE11. :) https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_Operator